internal/provider/provider.go

```diff
@@ -42,6 +42,17 @@ func (p *tlspcProvider) Metadata(ctx context.Context, req provider.MetadataReque
 
 func (p *tlspcProvider) Schema(ctx context.Context, req provider.SchemaRequest, resp *provider.SchemaResponse) {
 	resp.Schema = schema.Schema{
+		MarkdownDescription: `
+## Provider for the Venafi TLS Protect Cloud Platform
+
+This provider allows you to manage resources within the Venafi TLS Protect Cloud Platform.
+It's at an early stage of development; for production workloads, please ensure that versions are locked and upgrades considered to avoid breaking changes.
+
+### Usage
+
+We recommend that you create a custom user with the permissions required to manage the necessary resources, and use this user for performing terraform operations.
```

Comment:
Do we want to call out the Platform Administrator role as recommended?
Maybe include link to the roles definitions: https://docs.venafi.cloud/vaas/user-management/about-user-roles/
Or does the creation of `tlspc_plugin` resources need higher level admin perms. We could call that out here.. or maybe add a "lowest permission level: requirement to each resource?

Reply:
I was pretty much leaving it as an exercise for the reader. I don't really want to try to have to maintain a list of permission mappings. Probably sensible to at least link to the official docs, though.